developer-docs-site/docs/tutorials/run-a-fullnode-on-gcp.md

```diff
@@ -12,15 +12,67 @@ This tutorial explains how to configure and deploy a public FullNode to connect
 >
 
 ## Prerequisites
-Before you get started with this tutorial, install the required dependencies and get familiar with the toolings:
+You can run the commands in this guide to deploy your full node on Google Kubernetes Engine from any machine you want. From a [VM on GCP](https://cloud.google.com/compute), [Google Cloud Shell](https://cloud.google.com/shell), or even your personal computer.
+
+The following packages come pre-installed with Cloud Shell. Make sure to review the [documentation around ephermability](https://cloud.google.com/shell/docs/using-cloud-shell#choosing_ephemeral_mode) if you choose to use Cloud Shell. But if you are running the installation remotely from another machine, you need to install:
 * Terraform 1.1.7: https://www.terraform.io/downloads.html
 * Kubernetes cli: https://kubernetes.io/docs/tasks/tools/
 * Google Cloud cli: https://cloud.google.com/sdk/docs/install-sdk
 
+For a Debian distribution, you can run the following .sh file:
```

Comment:
Since we won't be able to provide a script for every OS, I think maybe we just remove this script, and let people figure out how to download those dependencies based on their OS?